.github/workflows/validate.yml

```diff
@@ -271,6 +271,71 @@ jobs:
           EXTRA_GHC: "/opt/ghc/${{ matrix.extra-ghc }}/bin/ghc-${{ matrix.extra-ghc }}"
         run: sh validate.sh ${{ env.COMMON_FLAGS }} --lib-only -s lib-suite-extras --extra-hc ${{ env.EXTRA_GHC }}
 
+  validate-alpine:
+    name: Validate statically linked alpine ghc-${{ matrix.ghc }}
```

Comment:
It doesn't actually validate it, it just builds the executable, right? It should say Build then.

Reply:
Good point! I've updated it now